tests/slurm_plugin/slurm_resources/test_slurm_resources.py

```diff
@@ -1105,7 +1105,7 @@ def test_slurm_node_is_powering_down_with_nodeaddr(node, expected_result):
         ),
         (
             StaticNode("queue1-st-c5xlarge-1", "ip-1", "hostname", "IDLE+CLOUD+POWER", "queue1"),
-            EC2Instance("id-1", "ip-1", "hostname", datetime(2020, 1, 1, 0, 0, 0)),
+            EC2Instance("id-1", "ip-1", "hostname", {"ip-1"}, datetime(2020, 1, 1, 0, 0, 0)),
```

Comment:
Nit-pick: Can we add more than one private-ip in `tests/slurm_plugin/slurm_resources/test_slurm_resources.py` too